content/en/docs/examples/virtual-machines/single-network/index.md

```diff
@@ -15,251 +15,102 @@ owner: istio/wg-environments-maintainers
-          -client -host spiffee://cluster.local/vm/vmname --out-priv key.pem --out-cert cert-chain.pem  -mode citadel
-    $ kubectl -n istio-system get cm istio-ca-root-cert -o jsonpath='{.data.root-cert\.pem}' > root-cert.pem
+    $ curl -v localhost:15000/clusters | grep httpbin
     {{< /text >}}
 
-### Setting up the VM
+This shows endpoints for `httpbin`:
 
-Next, run the following commands on each machine that you want to add to the mesh:
-
-1.  Copy the previously created `cluster.env` and `*.pem` files to the VM. For example:
-
-    {{< text bash >}}
-    $ export GCE_NAME="your-gce-instance"
-    $ gcloud compute scp --project=${MY_PROJECT} --zone=${MY_ZONE} {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
-    {{< /text >}}
-
-1.  Install the Debian package with the Envoy sidecar.
-
-    {{< text bash >}}
-    $ gcloud compute ssh --project=${MY_PROJECT} --zone=${MY_ZONE} "${GCE_NAME}"
-    $ curl -L https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ sudo dpkg -i istio-sidecar.deb
+    {{< text text >}}
+    outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_active::1
+    outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_connect_fail::0
+    outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_total::1
+    outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::rq_active::0
     {{< /text >}}
 
-1.  Add the IP address of the Istiod to `/etc/hosts`. Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the IP address.
-The following example updates the `/etc/hosts` file with the Istiod address:
-
-    {{< text bash >}}
-    $ echo "${IstiodIP} istiod.istio-system.svc" | sudo tee -a /etc/hosts
-    {{< /text >}}
+The IP `34.72.46.113` in this case is the pod IP address of the httpbin endpoint.
 
-   A better options is to configure the DNS resolver of the VM to resolve the address, using a split-DNS server. Using
-   /etc/hosts is an easy to use example. It is also possible to use a real DNS and certificate for Istiod, this is beyond
-   the scope of this document.
+### Send requests from virtual machine workloads to Kubernetes services
 
-1.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs/`.
+You can send traffic to `httpbin.default.svc.cluster.local` and get a response from the server. You must configure DNS in `/etc/hosts` to map the `httpbin.default.svc.cluster.local` domain name to an IP, or the IP will not resolve. In this case, the IP should be an IP that is routed over the single network using L3 connectivity. You should use the IP of the service in the Kubernetes cluster.
 
     {{< text bash >}}
-    $ sudo mkdir -p /etc/certs
-    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
+    $ curl -v httpbin.default.svc.cluster.local:8000/headers
     {{< /text >}}
```

Comment:
unindent

Reply:
thanks @frankbu  - missed this originally.